src/utils/addresses.ts

```diff
@@ -2,7 +2,7 @@ const getPartiallyHiddenEthereumAddress = (ethereumAddress: string): string => {
   const firstAddressSlice = ethereumAddress.slice(0, 6);
   const secondAddressSlice = ethereumAddress.slice(ethereumAddress.length - 4);
 
-  return `${firstAddressSlice} ･･･ ${secondAddressSlice}`;
+  return `${firstAddressSlice}･･･${secondAddressSlice}`;
```

Comment:
I've removed them because I did not see them [here](https://www.figma.com/file/KsuEveavWfON3Y2sgf96Jf/Polygon-Hermez-Bridge?node-id=2598%3A720):

<img width="400px" src="https://user-images.githubusercontent.com/15896177/183461698-ba9c18f4-e724-4298-a817-bcffbb879942.png">

Reply:
They are both different, the first one displays 5 characters before and after the dots while the second one just displays four. Also the dots are centered in the second one but not in the first one. The idea of each one is different, the first one tries to do an ellipsis while the second one tries to kinda "hide" some part of your address, so I would keep them different.

Reply:
OMG, do you mean the design team deliberately made this look differently? I never would have thought that was the case!

Reply:
I have asked the design team and although they had not done it consciously, they liked the idea you proposed of differentiating a shortened address from a hidden address. It's now implemented this way.